contracts/seed/Seed.sol

```diff
@@ -221,10 +221,12 @@ contract Seed {
             vestingStartTime = block.timestamp;
         }
 
-        _addFunder(
-            msg.sender,
-            (funders[msg.sender].fundingAmount + _fundingAmount)  // Previous Funding Amount + new funding amount
-            );
+        //functionality of addFunder
+        if (funders[msg.sender].fundingAmount==0) {
+            totalFunderCount++;
```

Comment:
Btw won't this approach break the contract a bit? Imagine, we have a funder, then he withdraws his tokens and his `funders[msg.sender].fundingAmount` becomes 0. If this exact funder will fund seed once again it will add a funder, even though he wasn't actually a new one..

Reply:
ohh yes, didn't think of this scenario. Well do we really need count of funder? As we aren't using it anywhere.

Reply:
The funder address is indexed, so if we won't count of funders, we can query the events for unique addresses

Reply:
There are few solutions for this problem, events is one of them. We can also have a mapping of funders or a structure that will store the data about the funder state. We can also remove a funder(decrement `totalFundCount`) when we are withdrawing funds.

Reply:
let's decrement totalFunderCount. Sounds good?

Reply:
Yep